groovy/groovy.support/src/org/netbeans/modules/groovy/support/actions/singlefilerun/SingleGroovySourceRunActionProvider.java

```diff
@@ -147,7 +152,7 @@ private Process setupProcess(String port) throws InterruptedException {
             commandList.add("-cp"); // NOI18N
             // TODO can we add here some heuristic, from which folder and appropriate path the script can be run?
             // For example we can look for the `src` or `src/groovy` fodler, from which the script can be probably run. 
-            commandList.add(".:" + groovyJar.getAbsolutePath());    //NOI18N
```

Comment:
nitpick: no need for `new String[]`, there's `CharSequence...` as 2+ `join` params :)

Reply:
Replaced. Thanks